ledger-api/grpc-definitions/com/daml/ledger/api/v1/ledger_identity_service.proto

```diff
@@ -14,6 +14,13 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 
 // Allows clients to verify that the server they are communicating with exposes the ledger they wish to operate on.
 // Note that every ledger has a unique ID.
+//
+// When the participant is run in mode requiring authentication, all the calls
+// in this interface will respond with ``UNAUTHENTICATED``, if the caller fails
+// to provide a valid access token, and will respond with ``PERMISSION_DENIED``, if
+// the claims in the token are insufficient to perform a given operation.
+// Subsequently, only specific errors of individual calls not related to
+// authorization will be described.
```

Comment:
I think this is superfluous but either it should be added to every service or to none.

Reply:
Yes, you're right. I took the authorisation information from already available description from `admin` services. I'll remove this annotation from those services as well.